.github/workflows/ci.yml

```diff
@@ -282,6 +282,10 @@ jobs:
           # Finally, run the tests
           pnpm --filter="${{ matrix.projectName }}" ${{ matrix.command }} $lastRunFlag
 
+      - name: 'Print debug log'
```

Comment:
@mikachan : can we get some context on this change, please? I'm wondering if this should be part of any existing artifact (steps below) or if it's a temporary troubleshooting measure related to the PR itself.

Reply:
It's just a temporary solution to try to debug why all the block e2e tests are failing on this PR. I will remove it before merging, although it doesn't seem to be helping me much anyway! I assume I've added something to cause a fatal error so the e2e tests can't run, but I'm unable to reproduce locally and it's difficult to debug, which is why I wanted to see the debug log.

Reply:
Gotcha, thanks for clarifying! If the log can be of use (it doesn't sound like it's helping, though), feel free to include it with other job run artifacts.